apps/web/src/pages/templates/components/VariantsPage.tsx

```diff
@@ -85,10 +228,21 @@ export function VariantsPage() {
             variant={step}
             nodeType="variantRoot"
             data-test-id="variant-root-card"
+            errorMessage={
+              errorState?.variantIndex === variants.length
+                ? errorState.errorMessage
+                : missingProviderError?.errorMsg ?? variantsErrorMap?.[`variant-${variants.length}`]
+            }
+            isActiveError={errorState?.variantIndex === variants.length}
+            nodeErrorType={errorState?.variantIndex === variants.length ? errorState.errorType : undefined}
           />
         )}
         {isScrollable && <FloatingButton isUp={scrollPosition.y > 0} onClick={scrollTo} />}
       </ScrollArea>
     </VariantsListSidebar>
   );
 }
+
+const ChevronPlainUp = styled(ChevronPlainDown)`
+  transform: rotate(180deg);
+`;
```

Comment:
Could we create a `ChevronPlainDown` in the design system instead? This seems like something we'll want to use through the UI.